velox/docs/develop/types.rst

```diff
@@ -73,6 +73,27 @@ Examples:
 * Timestamp(-5000*24*60*60 - 1000, 123456) represents 1956-04-24 07:43:20.000123456
   (5000 days 1000 seconds before epoch plus 123456 nanoseconds).
 
+Floating point types (REAL, DOUBLE) have special values negative/positive infinity and
+not-a-number (NaN).
+
+For NaN the semantics are different than the standard floating point semantics:
+
+* NaN = NaN returns true.
+* NaN is treated as a normal value in join and group-by keys.
```

Comment:
What does it mean "treated as a normal value"?

Reply:
It means as a value that is comparable according to the special rules laid out as opposed to NULL, for example.